functions/concepts/src/main/java/com/example/functions/concepts/FileSystem.java

```diff
@@ -34,9 +34,9 @@ public void service(HttpRequest request, HttpResponse response)
     File currentDirectory = new File(".");
     File[] files = currentDirectory.listFiles();
     BufferedWriter writer = response.getWriter();
```

Comment:
Consider using a `PrintWriter`, e.g.,

```java
PrintWriter writer = new PrintWriter(response.getWriter());
writer.println(...)
```

Reply:
`response.getWriter()` expects a `BufferedWriter`, not a `PrintWriter`.

(I tried using `new BufferedWriter(printWriter)`, but those stubs didn't produce any output in the tests.)